### PR TITLE
[ICP-6881] Added supported modes to LUX KONOz Thermostat

### PR DESCRIPTION
**Pull request by PKacprowiczS**
@greens @MAblewiczS @MMateusiakS

### CODE REVIEW

**Review by greens — DISMISSED** [review]

**Comment by greens** [comment]
Looks good pending verification.

File: devicetypes/smartthings/zigbee-thermostat.src/zigbee-thermostat.groovy [hunk 1]
```diff
@@ -172,6 +173,7 @@ private parseAttrMessage(description) {
 			log.debug "FAN MODE"
 			map.name = "thermostatFanMode"
 			map.value = FAN_MODE_MAP[it.value]
+			map.data = [supportedThermostatModes: state.supportedFanModes]
```
Comment (greens) [inline devicetypes/smartthings/zigbee-thermostat.src/zigbee-thermostat.groovy]: Ah. This should say `supportedThermostatFanModes`.

Comment (PKacprowiczS) [reply devicetypes/smartthings/zigbee-thermostat.src/zigbee-thermostat.groovy]: Oh jeez...

**Comment by greens** [comment]
I think you need to define `setThermostatFanMode(String value)`.

**Review by greens — APPROVED** [review]

**Pull request merged**